src/ResourceManager/Compute/Commands.Compute/Models/PSOperationResponse.cs

```diff
@@ -0,0 +1,27 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+using System.Net;
+
+namespace Microsoft.Azure.Commands.Compute.Models
+{
+    public class PSOperationResponse
```

Comment:
May I know what's the specific wrong format about? If something is wrong, it's better to fix it correctly, isn't it?